jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/internal/ExtensionStack.java

```diff
@@ -122,8 +123,6 @@ public void negotiate(DecoratedObjectFactory objectFactory, ByteBufferPool buffe
 
         this.extensions = new ArrayList<>();
 
-        String rsvClaims[] = new String[3];
-
         for (ExtensionConfig config : negotiatedConfigs)
         {
             Extension ext;
```

Comment:
the debug logs below (a fair way) should be protected with `if (LOG.isDebugEnabled ...`